router/projects-router.js

```diff
@@ -0,0 +1,54 @@
+const router = require("express").Router();
+
+const Projects = require("./projects-model");
+const Actions = require("./actions-model");
+const db = require("../data/dbConfig");
+
+router.get("/projects/", (req, res) => {
+  Projects.find()
+    .then(projects => {
+      res.status(200).json(projects);
+    })
+    .catch(err => {
+      res
+        .status(500)
+        .json({ message: "There was an error retrieving the project." });
+    });
+});
+
+router.get("/projects/:id/", (req, res) => {
+  Projects.findById(req.params.id)
+    .then(projects => {
+      Actions.find()
+        .where({ project_id: req.params.id })
+        .then(actions => {
+          projects.actions = actions;
+          return res.status(200).json(projects);
+        });
+    })
+    .catch(err => {
+      res
+        .status(500)
+        .json({ message: "There was an error retrieving the project." });
+    });
+});
+
+router.post("/projects/", async (req, res) => {
+  const project = req.body;
+  if (project.name) {
+    try {
+      const inserted = await Projects.add(project);
+      res.status(201).json(inserted);
+    } catch (error) {
+      res
+        .status(500)
+        .json({ message: "There was an error creating the project." });
+    }
+  } else {
+    res
+      .status(400)
+      .json({ message: "Please provide the name of the waterpark." });
+  }
+});
+
```

Comment:
You did an excellent job Brock, I'm really impressed with your progress, your code is really clean, it was super easy for me to readi and understand all was going on and I really enjoyed reviewing it. Wish you the bet in upcoming Sprints!